src/main/java/org/codehaus/mojo/clirr/ClirrClassFilter.java

```diff
@@ -82,6 +100,22 @@ public boolean isSelected( JavaClass javaClass )
                     result = !SelectorUtils.matchPath( excludes[i], path );
                 }
             }
+
+            if (excludeAnnotated != null && javaClass.getAnnotationEntries() != null)
+            {
+                for (AnnotationEntry annotationEntry : javaClass.getAnnotationEntries())
```

Comment:
Let's cache `javaClass.getAnnotationEntries()` in a local variable. No need to call it twice.

Reply:
@martin-g Done.